book/website/reproducible-research/licensing/licensing-compatibility.md

```diff
@@ -248,6 +250,7 @@ If it did not include OpenIFS, it would have to be distributed under the GPLv3,
 #### Can we work on this privately, without distributing anything?
 
 The GPL allows making private modifications of software covered by it, with no restrictions, provided the changed software is not distributed at all.
+In the case of the AGPL distribution would include running a server interacted with in some way by users.
```

Comment:
```suggestion
In the case of the AGPL, distribution includes users interacting with the software running a server.
And so, if the software is made available to use that way, the source must be made available under the terms of the AGPL.
```

Adding a sentence to clarify what this means.
Is it true that using AGPL software over a network counts as distribution? I (without reading the license properly) assumed it was that _users_ count if they use the software over a network, and users are entitled to the source.

Reply:
I should double check but yes my understanding is that if you user AGPL software over a network, you are entitled to a copy of the source. I need to clarify the phrasing to convey the obligations of someone running AGPL software on a server.

Reply:
I checked and as far as I can tell if you interact with a service the code for which is AGPL over a network it triggers the equivalent requirements as if you had sent me the server binary - in either case you have to make the source available to me. I tried rewording for clarity in [622da27](https://github.com/the-turing-way/the-turing-way/pull/3142/commits/622da27873f870341ac993f16c4981fea3211ac0) it now reads: "In the case of the AGPL, running a server interacted with in some way by users over a network is equivalent to distribution under the GPL and you would be required to provide any users with the source code."